common/schemas/keyman-touch-layout/README.md

```diff
@@ -1,11 +1,11 @@
 # keyman-touch-layout
 
-Documentation at https://help.keyman.com/developer/...TODO...
+Documentation at https://github.com/keymanapp/keyman/tree/master/common/schemas/keyman-touch-layout/README.md
 
 * Primary version:
   * https://github.com/keymanapp/keyman/tree/master/common/schemas/keyman-touch-layout
 * Deployed to:
-  * N/A
+  * https://github.com/keymanapp/api.keyman.com/tree/master/schemas/keyman-touch-layout
```

Comment:
I saw the clean spec deployed to
https://github.com/keymanapp/api.keyman.com/tree/master/schemas/keyman-touch-layout/2.1.1

Where does that come from?

Reply:
> Where does that come from?

The clean spec is also in the keyman repo.